src/dapp-tests/integration/tests.sh

```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env bash
 
-set -e
+set -ex
```

Comment:
Is this nice to have on CI outputs? Or locally?

Reply:
It's nice on ci outputs!